srcs/utils/get_options.c

```diff
@@ -0,0 +1,29 @@
+# include "libft.h"
+
+char	*get_options_core(int ac, char **av)
+{
+	char	*arg;
+	int		i;
+	int		j;
+
+	if (ac <= 1 || !av || ft_tablen(av) <= 1)
+		return (NULL);
+	arg = NULL;
+	i = 0;
+	while (++i < ac && av[i])
+	{
+		if (ft_strlen(av[i]) >= 2 && av[i][0] == '-')
+		{
+			j = 0;
+			while (av[i][++j])
+			{
+				if (ft_isalpha(av[i][j]))
```

Comment:
The first -- argument that is not an option-argument should be accepted as a delimiter indicating the end of options. Any following arguments should be treated as operands, even if they begin with the '-' character.